docs/server/tutorials/extensions.md

```diff
@@ -4,63 +4,43 @@ title: Sourcegraph extensions - setup and usage instructions (alpha)
 permalink: docs/extensions
 ---
 
-Sourcegraph is getting an extension API that makes easy to add new features and information to Sourcegraph, GitHub, and other code hosts and review tools that our browser extensions integrate with. The [Sourcegraph extension API](https://github.com/sourcegraph/sourcegraph-extension-api) allows extensions to provide hovers, definitions, references, line decorations, buttons, menu items, and other similar contributions. For more information, see the open-source [sourcegraph-extension-api repository](https://github.com/sourcegraph/sourcegraph-extension-api).
+Sourcegraph is getting an extension API that makes it easy to add new features and information to Sourcegraph, GitHub, and other code hosts and review tools that our browser extensions integrate with. The [Sourcegraph extension API](https://github.com/sourcegraph/sourcegraph-extension-api) allows extensions to provide hovers, definitions, references, line decorations, buttons, menu items, and other similar contributions. For more information, see the open-source [sourcegraph-extension-api repository](https://github.com/sourcegraph/sourcegraph-extension-api).
```

Comment:
Maybe change "is getting" to "has"? Not sure if you intend to present this as a future thing (since we say it's in alpha below) or current. Either is fine.

Reply:
:+1: to "has", since they're here already